katdal/datasources.py

```diff
@@ -164,7 +164,8 @@ def name(self):
         return name
 
 
-def view_capture_stream(telstate, capture_block_id=None, stream_name=None):
+def view_capture_stream(telstate, capture_block_id=None, stream_name=None,
+                        **kwargs):
```

Comment:
kwargs needs to be added to the docstring. It doesn't seem to be used anywhere - is the intent that it's just there to swallow arguments that aren't `capture_block_id` or `stream_name`?

Reply:
Correct. I was going to update the docs some day... Let's make it today then.